api/nakadi-event-bus-api.yaml

```diff
@@ -888,6 +888,7 @@ definitions:
               Timestamp of creation of the Event generated by the producer.
             type: string
             format: date-time
+            example: '1996-12-19T16:39:57-08:00'
```

Comment:
Unfortunately, it looks like that Joda implementation is more lenient than ours. I tested with the string `"1996-03-15T16:39:57+0100"` and it was accepted by their implementation.

But the RFC  states that the `:` is mandatory. So I'll stay with our implementation, by now. We should be comply with RFC 3339 and not ISO 8601, which is not intended "for internet".